cloud/disk_manager/internal/pkg/dataplane/nbs/source.go

```diff
@@ -249,11 +249,14 @@ func NewDiskSource(
 	dontReadFromCheckpoint bool,
 ) (dataplane_common.Source, error) {
 
-	if len(proxyDiskID) == 0 {
-		proxyDiskID = diskID
+	var session *nbs.Session
+	var err error
+	if len(proxyDiskID) != 0 {
```

Comment:
Here we pass proxyDiskID == diskID https://github.com/ydb-platform/nbs/blob/b4ff2c5e13e4a2a55cb15064785ce82688e27df3/cloud/disk_manager/internal/pkg/dataplane/transfer_from_disk_to_disk_task.go#L59

Looks like it should be fixed